docs/frontend.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[frontend]]
+== Frontend support
+
+NOTE: Frontend support is an experimental feature not intended for production usage at the moment. 
+
+This section describes features specifically designed to support frontend application monitoring
+and a guide on how to enable experimental frontend support. 
+
+[[frontend-enable]]
+[float]
+=== Enable Frontend Support
+To try out experimental frontend support, set the `apm-server.frontend.enabled` to `true`.
+See https://github.com/elastic/apm-server/blob/{doc-branch}/apm-server.reference.yml[`apm-server.reference.yml`] for more configuration options.
+
+[[sourcemap]]
+=== Source Maps
+It is common practice to compress and minify client side javascript code for several reasons, e.g. performance gain. 
```

Comment:
I suggest: "It is common practice to minify client-side JavaScript code for several reasons, e.g. performance gain."